website/pages/docs/job-specification/hcl2/index.mdx

```diff
@@ -111,11 +111,50 @@ meta {
 Additionally, block attributes must be [HCL2 valid identifiers](https://github.com/hashicorp/hcl/blob/v2.8.0/hclsyntax/spec.md#identifiers).
 Generally, identifiers may only contain letters, numbers, underscore `_`,
 or a dash `-`, and start with a letter. Notable,
-[`meta`](https://www.nomadproject.io/docs/job-specification/meta), and
-[`env`](https://www.nomadproject.io/docs/job-specification/env) keys may not
+[`meta`](/docs/job-specification/meta), and
+[`env`](/docs/job-specification/env) keys may not
 contain other symbols (e.g. `.`, `#`).
 
-### Multiline "here doc" strings
+Task Driver config fields may require extra attention: If they contain invalid
+identifiers. For example, docker [`sysctl`](/docs/drivers/docker#sysctl) must
+use the map assignment syntax if the keys aren't valid:
+
+```hcl
+sysctl = {
+  "net.core.somaxconn/docs/drivers/docker#sysctl" = "16384"
+}
+```
+
+Additionally, Task Driver config fields may not nest block syntax within an
```

Comment:
We don't typically capitalize "task driver" like a proper noun except in section titles:

```suggestion
Additionally, task driver config fields may not nest block syntax within an
```